LocalPackages/DataBrokerProtection/Sources/DataBrokerProtection/Operations/DataBrokerProfileQueryOperationManager.swift

```diff
@@ -389,6 +396,18 @@ struct DataBrokerProfileQueryOperationManager: OperationsManager {
                                                      schedulingConfig: schedulingConfig)
         }
 
+    internal func increaseAttemptCountIfNeeded(database: DataBrokerProtectionRepository,
+                                               brokerId: Int64,
+                                               profileQueryId: Int64,
+                                               extractedProfileId: Int64) throws {
```

Comment:
Can this function be private? I don’t see it called from any tests. Also I’m seeing Swiftlint errors for parameter alignment here.